test/Calibur.execute.t.sol

```diff
@@ -818,12 +813,12 @@ contract CaliburExecuteTest is TokenHandler, HookHandler, ExecuteFixtures, Deleg
 
         bytes memory wrappedSignature = abi.encode(signature, EMPTY_HOOK_DATA);
         signerAccount.execute(signedBatchedCall, wrappedSignature);
-        vm.snapshotGasLastCall("execute_BATCHED_CALL_opData_P256_singleCall");
+        vm.snapshotGasLastCall("execute_BATCHED_CALL_withSignature_P256_singleCall");
```

Comment:
nit: should we also not use BATCHED_CALL language in the snapshots when we are not using the 7821 entrypoint?